src/System.CommandLine.Suggest/ISuggestionStore.cs

```diff
@@ -6,7 +6,7 @@ namespace System.CommandLine.Suggest
 {
     public interface ISuggestionStore
```

Comment:
Should the interface be renamed too?

Reply:
I was leaving `dotnet-suggest` alone for now but might come back to it.